libcodechecker/analyze/ctu_manager.py

```diff
@@ -106,7 +106,7 @@ def generate_ast(triple_arch, action, source, config, env):
             pass
 
     cmd = ctu_triple_arch.get_compile_command(action, config, source)
-    cmd.extend(['-emit-ast', '-w', '-o', ast_path])
+    cmd.extend(['-emit-ast', '-D__clang_analyzer__', '-w', '-o', ast_path])
```

Comment:
Please add a short comment in the source code too why this extra flag is required.